src/styles/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: $bodyFont;
```

Comment:
Ensure that the `$bodyFont` variable is defined in your SCSS. If it's not defined, the compilation will fail. If this variable is defined in another file, make sure it's imported correctly.